src/Compilers/Core/Portable/Collections/SmallDictionary.cs

```diff
@@ -1,39 +1,36 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Text;
-using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis
 {
-    // Dictionary designed to hold small number of items.
-    // Compared to the regular Dictionary, average overhead per-item is roughly the same, but 
-    // unlike regular dictionary, this one is based on an AVL tree and as such does not require 
-    // rehashing when items are added.
-    // It does require rebalancing, but that is allocation-free.
-    //
-    // Major caveats:
-    //  1) There is no Remove method. (can be added, but we do not seem to use Remove that much)
-    //  2) foreach [keys|values|pairs] may allocate a small array.
-    //  3) Performance is no longer O(1). At a certain count it becomes slower than regular Dictionary.
-    //     In comparison to regular Dictionary on my machine:
-    //        On trivial number of elements (5 or so) it is more than 2x faster.
-    //        The break even count is about 120 elements for read and 55 for write operations (with unknown initial size).
-    //        At UShort.MaxValue elements, this dictionary is 6x slower to read and 4x slower to write
-    //
-    // Generally, this dictionary is a win if number of elements is small, not known beforehand or both.
-    //
-    // If the size of the dictionary is known at creation and it is likely to contain more than 10 elements, 
-    // then regular Dictionary is a better choice.
-    //
+    /// <summary>
```

Comment:
:exclamation: This is only a partial conversion to a documentation comment. Did you mean to convert the internal structure to its XML form as well?